compute_tools/src/http/api.rs

```diff
@@ -62,6 +63,18 @@ async fn routes(req: Request<Body>, compute: &Arc<ComputeNode>) -> Response<Body
             Response::new(Body::from(serde_json::to_string(&metrics).unwrap()))
         }
 
+        // Prometheus metrics
+        (&Method::GET, "/metrics") => {
+            info!("serving /metrics GET request");
+            match prometheus_metrics_handler(req).await {
```

Comment:
question for reviewers:

This is a default handler, used by proxy and other services. It will collect some extra process metrics. Is it ok (do we filter them out anywhere) or I need to write new handler to explicitly collect only 'installed_extensions' metric?

Reply:
Fixed. 
Now only one metric is collected.
If we will want to add more compute_ctl metrics, we will define compute registry.